lib/warningNotes.spec.ts

```diff
@@ -53,4 +53,57 @@ describe('warningNotesAPI', () => {
       expect(data).toEqual(['foo']);
     });
   });
+
+  describe('updateWarningNote', () => {
+    it('should work properly if the status is open', async () => {
```

Comment:
I think `should work properly...` is a little vague for a test title. What does 'work properly' mean? Instead I might expect something like: "it should respond to a PATCH request with a 200 OK and {some data} if the status is open".

Perhaps not one to resolve here and now, but something to think about. If you came back to this test in 6 months, would this name help you understand what it was testing?

Reply:
Yeah I agree on this, many of our tests already follow this naming convention so I kept it the same for consistency but I agree they should be more descriptive